scripts/generate-parser-feedback-test/index.ts

```diff
@@ -40,7 +40,41 @@ function appendToken(dest: Token[], token: Token): void {
     dest.push(token);
 }
 
-function collectParserTokens(html: string): ReturnType<typeof convertTokenToHtml5Lib>[] {
+function convertTokenToHtml5Lib(token: Token): HtmlLibToken {
```

Comment:
`convertTokenToHtml5Lib` was no longer needed in `test/utils/generate-tokenization-tests.ts`, and therefore moved here.